pom.xml

```diff
@@ -581,7 +581,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hive</groupId>
                 <artifactId>hive-apache</artifactId>
-                <version>1.2.2-2</version>
+                <version>3.0.0-2</version>
```

Comment:
Should we just jump directly to 3.0.0-3 in this PR or re-apply the changes from https://github.com/prestodb/presto/pull/15796 after this one is merged?

Reply:
Yeah I'll update it